src/test/java/nextstep/study/RestAssuredTest.java

```diff
@@ -0,0 +1,19 @@
+package nextstep.study;
+
+import io.restassured.RestAssured;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class RestAssuredTest {
+
+    @DisplayName("구글 페이지 접근 테스트")
+    @Test
+    void accessGoogle() {
+        RestAssured
+                .given()
+                .when()
+                    .get("https://google.com")
+                .then()
+                    .statusCode(200);
```

Comment:
Nit: 이미 잘 정의된 상수를 활용해보는 것도 좋을 것 같습니다.
```suggestion
                    .statusCode(HttpStatus.OK.value());
```